src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/ClanManager.java

```diff
@@ -1044,9 +1046,9 @@ public boolean purchaseResetKdr(Player player) {
         if (plugin.getPermissionsManager().hasEconomy()) {
             if (plugin.getPermissionsManager().playerHasMoney(player, price)) {
                 plugin.getPermissionsManager().playerChargeMoney(player, price);
-                player.sendMessage(RED + MessageFormat.format(lang("account.has.been.debited", player), price));
```

Comment:
Doesn't `player.spigot().sendMessage(new TextComponent(...))` work for you? 🤔

Reply:
> I didn't see any use in the whole project

Well, we haven't this necessity earlier then.
Use `player.spigot().sendMessage()`.
I don't remember when [Chat Components API](https://ci.md-5.net/job/BungeeCord/ws/chat/target/apidocs/net/md_5/bungee/api/chat/package-summary.html) was implemented, I mean, it may not work on 1.8.

There is one more way trough `player.sendRawMessage()`. 🤔

Reply:
sendMessage doesn't work inside prompt flows.
Options:
1. Send after the prompt finished (using a Bukkit runnable)
2. Return a MessagePrompt

Reply:
> sendMessage doesn't work inside prompt flows. Options:
> 
>     1. Send after the prompt finished (using a Bukkit runnable)
> 
>     2. Return a MessagePrompt


Hm, spigot() thing work for me 🤔 .
![изображение](https://user-images.githubusercontent.com/42711312/235677682-33a26380-29b1-4f4e-bd47-7171d1852831.png)